roles/kafka_broker/tasks/get_authorization_token.yml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Can we not reuse the tasks from common role ?

Reply:
it might work for broker, since the user is same. For other cp components like SR we need this file here. I will try out calling the common file from broker and delete this file if it works